Changelog.md

```diff
@@ -1,5 +1,36 @@
 ## Ignition Msgs 8.x
 
+### Ignition Msgs 8.3.0 (2022-03-24)
+1. backport NavSat message for bridge compatibility
+    * [Pull request #231](https://github.com/ignitionrobotics/ign-msgs/pull/231)
+
+1. Add docs to wheel_slip_parameters_cmd.proto and update entity field
+    * [Pull request #227](https://github.com/ignitionrobotics/ign-msgs/pull/227)
+
+1. Focal CI: static checkers and doxygen linters
+    * [Pull request #230](https://github.com/ignitionrobotics/ign-msgs/pull/230)
+
+1. Pose, twist and odometry with covariance msgs
+    * [Pull request #224](https://github.com/ignitionrobotics/ign-msgs/pull/224)
+
+1. Add wheel slip message definition
+    * [Pull request #205](https://github.com/ignitionrobotics/ign-msgs/pull/205)
+
+1. Add Ubuntu Jammy CI
+    * [Pull request #225](https://github.com/ignitionrobotics/ign-msgs/pull/225)
+
+1. Add missing include to PointCloudPackedUtils
+    * [Pull request #218](https://github.com/ignitionrobotics/ign-msgs/pull/218)
```

Comment:
this can be grouped with the other point cloud PR

* https://github.com/ignitionrobotics/ign-msgs/compare/ignition-msgs8_8.2.0...aditya/8.3.0#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R83-R85

Reply:
Done